documentation/monitoring.md

```diff
@@ -102,6 +140,14 @@ There are several other variables that can be changed depending on env requireme
 - alertmanager_app_mem - app memory limit (default 1G)
 - alertmanager_app_cpu - app cpu requests (default 1)
 
+The alertmanager urls corresponding to the various environments are
+### development | https://alertmanager.cluster[clusternumber].development.teacherservices.cloud/
```

Comment:
development is not required here, as the developers don't need it

Reply:
removed